### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by priom**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (ansermino) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Not relevant

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (ansermino) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Not relevant

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```
Comment (ansermino) [inline .github/ISSUE_TEMPLATE/bug_report.md]: A bit confused by the point of this section. Labels and assignees are available through github

**Comment by ansermino** [comment]
You might also add a check box at the bottom for `[ ] I plan to submit a PR for this issue`

**Comment by priom** [comment]
@ansermino i'm gonna close this PR and open a new one...seems like it was a boilerplate issue template

**Pull request closed without merging**